encoding/legacy-mb-schinese/gb18030/gb18030-encoder.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<meta charset=gb18030>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script src=resources/ranges.js></script>
+<div id=log></div>
```

Comment:
Is this `div` necessary these days? I thought testharness created it on the fly, but there may be other considerations.

Reply:
It's not necessary but it was there before.  I'll remove it.